src/cattr/converters.py

```diff
@@ -27,13 +27,17 @@
 )
 from .disambiguators import create_uniq_field_dis_func
 from .multistrategy_dispatch import MultiStrategyDispatch
+import attr
 
 
 NoneType = type(None)
 T = TypeVar("T")
 V = TypeVar("V")
 
 
+CATTRS_METADATA_KEY = 'cattrs_structure_key'
```

Comment:
I tried to follow the advise made here: http://www.attrs.org/en/stable/extending.html. I agree that it looks weird in my code to have an `cattr.ib`.

An alternative would be to use a more distinctive name like in the referenced example, e.g. `cattr.ib_mapped(..., src_key='mapped')`.

Of course, also this would work

```python
from cattr import CATTRS_METADATA_KEY

attrs.ib(..., metadata={CATTRS_METADATA_KEY: 'mapped'})
```
But it looks very ugly - at least `CATTRS_METADATA_KEY` should be more self explanatory + shorter.

What do you think @Tinche? Being active on `attrs` itself you certainly have a strong opinion.